template/src/ontology/Makefile.jinja2

```diff
@@ -1058,6 +1061,19 @@ $(TRANSLATIONSDIR)/%.babelon.json: $(TRANSLATIONSDIR)/%.babelon.tsv
 {% endif -%}
 {% endfor -%}
 
+{% if 'db' in project.export_formats -%}
+{% if project.use_context -%}
```

Comment:
What a great idea to piggy bag on `use_context`!!

Reply:
Better to have only one source of truth for the prefixes. ;)